packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -62,7 +72,7 @@
                   <img
                     v-if="imageContentObject[space.id]"
                     class="space-image oc-rounded-top"
-                    :src="'data:image/jpeg;base64,' + imageContentObject[space.id]"
+                    :src="'data:image/jpeg;base64,' + imageContentObject[space.id]['content']"
```

Comment:
is it guaranteed to be `data:image/jpeg`?

Reply:
definitely not, thx on it